src/utils/helper-functions/versions.ts

```diff
@@ -0,0 +1,23 @@
+import * as semver from 'semver'
+
+let ngrxEffectsVersion: string | undefined
+function getNgrxEffectsVersion(): string {
+  if (ngrxEffectsVersion) return ngrxEffectsVersion
+
+  try {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const effects = require('@ngrx/effects/schematics-core') as {
+      platformVersion: string
```

Comment:
Is it optional? If so:

```suggestion
      platformVersion?: string
```

If not, let's remove the `optional chaining` below.

Reply:
It's not optional in the last few versions of ngrx, before that, it wasn't exported 😅
That's why I wanted to be cautious

Reply:
Aha, in that case, I'd suggest the first option (marking it with `?`).